src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useContext, useEffect, useReducer } from "react"
+import { useDidMount } from "Utils/Hooks/useDidMount"
+import { hasFilters } from "./Utils/hasFilters"
+import { isDefaultFilter } from "./Utils/isDefaultFilter"
+import { rangeToTuple } from "./Utils/rangeToTuple"
+import { urlFragmentFromState } from "./Utils/urlFragmentFromState"
+
+export interface Filters {
```

Comment:
Using this same shape for both the props passed into the provider and the filters available to consumers might be problematic. `major_periods`, for example, should not be required to be passed into the provider, but it should be always included for the consumers (because it's an array, and it's just easier to deal with it as an empty array).